ynput/OpenPype
# colorspace: get_imageio_config signature change

## Changelog Description
Colorspace distribution API changes

## Additional info
Adding new args into `get_imageio_config` and implementing `get_current_context` and `get_template_data_with_names`

## Testing notes:
1. Make sure you are having enabled Global colorspace management
2. in Ayon or Openpype open Tray/Admin/Console
3. Run following code - and change the project name to yours
```
from openpype.pipeline import colorspace

project_name = "<< Your project name >>"
asset_name = "<< Your asset name >>"
task_name = "<< Your task name >>"

config_data = colorspace.get_imageio_config(
    project_name=project_name,
	asset_name=asset_name,
	task_name=task_name,
    host_name="nuke"
)
print(config_data)
```
4. Dictionary with config_data should be returned. 
5. In all hosts it should work as before.
